games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef.in

```diff
@@ -0,0 +1,38 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Artsoft-rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major M-BM- = @MAJOR@,
```

Comment:
On this line (and the line for "minor") there were 2 unprintable chars, so you still have to remove one "M-BM-"

You also have multiple spaces below, which should be replaced ny tabs.